api/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Fri Jan 08 13:49:35 EST 2021
```

Comment:
Probably revert this change, different IDEs seem to mutate this file in different ways